Doc/library/asyncio-eventloop.rst

```diff
@@ -47,9 +47,6 @@ an event loop:
    call_soon or similar API), this function will always return the
    running event loop.
 
-   If there is no running event loop set, the function will return
-   the result of the ``get_event_loop_policy().get_event_loop()`` call.
```

Comment:
This is still true.

`get_event_loop_policy().get_event_loop()` can return the loop that was set by `get_event_loop_policy().set_event_loop()`.